typescript/misc.d.ts

```diff
@@ -38,6 +40,30 @@ export interface UserProfile {
   phoneNumberVerified?: boolean
   /** The users list of groups from a SAML configuration */
   groups?: string[]
+  /** The user's address area/zip/post code */
+  areaCode?: string
+  /** The user's address */
+  address?: string
+  /** The user's city */
+  city?: string
+  /** The user's state */
+  state?: string
+  /** The user's country */
+  country?: string
+  /** The user's country calling code */
+  countryCallingCode?: string
+  /** The user's department */
+  department?: string
+  /** The user's division */
+  division?: string
+  /** The user's bargain */
+  bargain?: string
```

Comment:
I was under the impression Blake was pushing back on this

Reply:
He has altered the bargain.